src/linker/Linker/TypeReferenceExtensions.cs

```diff
@@ -61,10 +61,12 @@ public static StringBuilder GetDisplayNameWithoutNamespace (this TypeReference t
 					break;
 				}
 
-				type = type.DeclaringType;
-				if (type == null)
+				var nType = type.DeclaringType;
+				if (nType == null)
```

Comment:
Nit: why is this needed?

Reply:
I needed a separate variable that is nullable since the type is not. I've cleaned this up a bit.